src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -16,6 +16,19 @@
   top: 0;
   transition: bottom $animation-duration-short ease-out;
 
+  * {
+    // This aims to prevent possibly conflicting style definitions inherited
+    // from target applications which can break subtitles styling. It's still possible
+    // to override this with selector of higher priority score.
+    all: unset;
```

Comment:
AFAIK IE doesn't support this? 🤔 Do we care about IE or is there a different possibility for IE? 🤔

Reply:
Yep true, we dediced not to care for IE in this case. It would require quite a lot of lines just to cover the reset. And its also just a safety check.

Reply:
Ok please add a comment stating that this doesn't work on IE to the existing comment.